SEED-platform/seed
# 2512 feat/add property display setting

#### Any background context you want to provide?

#### What's this PR do?
Adds an organization setting to change which field is used to display a property's name.

#### How should this be manually tested?
See the org settings page, change the setting and then view a property detail page.

#### What are the relevant tickets?
https://github.com/SEED-platform/seed/issues/2512

#### Screenshots (if appropriate)
(see ticket)
